src/storage/kv.rs

```diff
@@ -33,7 +33,11 @@ impl LogKeyValueStore {
         let db_version_minor_str = env!("CARGO_PKG_VERSION_MINOR");
         let db_version_revise_str = env!("CARGO_PKG_VERSION_PATCH");
 
-        let db_version = vec![db_version_major_str, db_version_minor_str, db_version_revise_str];
+        let db_version = vec![
+            db_version_major_str,
+            db_version_minor_str,
+            db_version_revise_str,
+        ];
```

Comment:
Uh...I thought our code pass `cargo fmt` many times. Why do we have these formatting changes again and again.

Reply:
hmm, I am not sure it is cargo fmt's fault, I will keep an eye on it.